frontend/src/views/DistanceCharts.vue

```diff
@@ -60,8 +60,9 @@ const chartStore = useChartsStore()
 const hydrologicStore = useHydrologicStore()
 
 // TODO:nodes on tab switch, update distance chart data by query nodes
-let nodeVariables = hydrologicStore.getPlottableSwotVariables('node')
-let varTab = ref(nodeVariables[0])
+//let nodeVariables = hydrologicStore.getPlottableSwotVariables('node')
```

Comment:
remove `getPlottableSwotVariables()` function from the `hydrologic.js` store if we aren't going to use it anymore. And remove this commented line.

Reply:
done.

Reply:
thanks!